metriq_gym/run.py

```diff
@@ -21,19 +24,61 @@ def main():
 
         handler = HANDLERS[JobType(params["benchmark_name"])](args, params, job_manager)
         handler.dispatch_handler()
+
     elif args.action == "poll":
         job_id = args.job_id
         job = job_manager.get_job(job_id)
         job_type = job["job_type"]
 
         handler = HANDLERS[JobType(job_type)](args, None, job_manager)
         handler.poll_handler()
+
+    elif args.action == "list-jobs":
+        list_jobs(args)
+
     else:
-        logging.error("Invalid action specified. Use 'dispatch' or 'poll'.")
+        logging.error("Invalid action specified. Run with --help for usage information.")
         return 1
 
     return 0
 
 
+def list_jobs(args: argparse.Namespace) -> int:
+    """List jobs recorded in the jobs file.
+
+    Args:
+        args: Parsed arguments.
+    Returns:
+        Return code.
+    """
+    # Read jobs from the file.
+    try:
+        with open(args.jobs_file, "r") as f:
+            jobs = [json.loads(line) for line in f]
+    except FileNotFoundError:
+        logging.error(f"Jobs file not found: {args.jobs_file}")
+        return 1
+    except json.JSONDecodeError:
+        logging.error(f"Error reading jobs from file: {args.jobs_file}")
+        return 1
```

Comment:
this is exactly what `JobManager.get_jobs()` does 🤷

Reply:
Yep, completely redundant. The `JobManager` now does effectively this behavior instead here  71936ee